app/assets/v2/css/profile.css

```diff
@@ -365,3 +372,28 @@ a:hover .avatar_edit .avatar_edit_text {
 .no-profile-textcolor {
   color: #666666;
 }
+
+.profile-banner {
+  position: absolute;
+  top: 25vh;
+  color: #ffffff;
+  z-index: 99;
+  text-align: center;
+  font-size: 1.7rem;
+  font-weight: 700;
+  width: 100vw;
+  visibility: hidden;
```

Comment:
Stick to using the predefined font classes ? 
https://github.com/gitcoinco/web/blob/master/app/assets/v2/css/gitcoin.css#L144-L170